Bottom: 8a6a16419009fe7f6ea98593def9c67d5ee46d25
Top:    68f9c4b8d2dcb144d86165fa202cee3958909eec
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-21 11:53:11 +0000

Refresh of move-more-rendering-to-world-c

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 490ccc4..431b722 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -258,12 +258,12 @@ ghid_note_event_location (GdkEventButton * ev)
   ghid_unproject_to_z_plane (event_x, event_y, global_depth,
                              &gport->view_x, &gport->view_y);
 #else
-  gport->view_x = event_x * gport->zoom + gport->view_x0;
-  gport->view_y = event_y * gport->zoom + gport->view_y0;
+  gport->view_x = SIDE_X (event_x * gport->zoom + gport->view_x0);
+  gport->view_y = SIDE_Y (event_y * gport->zoom + gport->view_y0);
 #endif
 
-  moved = MoveCrosshairAbsolute (SIDE_X (gport->view_x), 
-				 SIDE_Y (gport->view_y));
+  moved = MoveCrosshairAbsolute (gport->view_x,
+				 gport->view_y);
   if (moved)
     {
       AdjustAttachedObjects ();
